### PR TITLE
[LPT] [CPU] NormalizeL2 transformation

### PR DESCRIPTION
**Pull request by eshoguli**
User story: 31416: [LPT] NormalizeL2 layer transformation

### CODE REVIEW

**Comment by dmitry-gorokhov** [comment]
@eshoguli  In general LGTM.
I merged mkl-dnn related changes on v0.21_for_ie_master branch so I updated commit hash here.
Please also fix conflicts in LPT tests.

**Comment by eshoguli** [comment]
@dmitry-gorokhov done, rebased

**Comment by dmitry-gorokhov** [comment]
> @dmitry-gorokhov done, rebased

And now CI is red

File: inference-engine/src/cldnn_engine/cldnn_program.cpp [hunk 1]
```diff
@@ -238,7 +238,8 @@ Program::Program(InferenceEngine::ICNNNetwork& network, std::shared_ptr<const cl
 
         auto transforms = LowPrecisionTransformer::getAllTransformations(params)
                 .add<FullyConnectedTransformation>(LayerTransformation::Params(params).setSupportAsymmetricQuantization(false), "FullyConnected")
-                .add<FullyConnectedTransformation>(LayerTransformation::Params(params).setSupportAsymmetricQuantization(false), "GEMM");
+                .add<FullyConnectedTransformation>(LayerTransformation::Params(params).setSupportAsymmetricQuantization(false), "GEMM")
+                .removeTransformations("Normalize");
```
Comment (echuraev) [inline inference-engine/src/cldnn_engine/cldnn_program.cpp:242]: Fix for NormalizeL2 was merged to master (#720). Please remove this line.

Comment (eshoguli) [reply inference-engine/src/cldnn_engine/cldnn_program.cpp]: done, thanks!

**Comment by dmitry-gorokhov** [comment]
@eshoguli Please resolve conflicts.

**Pull request merged**